src/Component/CategoryDetail/Component/MdChoice/Pagination.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from "react";
+import "./Pagination.scss";
+
+class Pagination extends Component {
+  render() {
+    const { fetchProduct, activePage } = this.props;
+
+    return (
+      <div className="Pagination">
+        <button
+          data-idx="0"
+          onClick={fetchProduct}
+          className={activePage === "0" ? "active" : null}
```

Comment:
boolean값을 이용해 아래와 같이 바꾸어 주실 수 있습니다 🙌🏻
```suggestion
          className={!activePage ? "active" : null}
```

Reply:
음 ㅜㅜ근데 이건 값이 같은지 확인하는건데도
저렇게 불리언 값으로 해야할까요? ㅜㅜ